docs/mobile-apps/features/audio-capture.md

```diff
@@ -70,23 +70,27 @@ If your apps's manifest.xml file defines `android:allowAudioPlaybackCapture="fal
 
 :::
 
-## Using TalkBack on Android Private Devices
+## Enabling TalkBack/Voiceover on your Devices
 
-To utilize TalkBack on your Android private device, make sure you have a device with TalkBack enabled (Android 10 and above, Google Pixel or Samsung devices). Follow these steps:
+To utilize TalkBack on your iOS/Android devices, make sure you have access to supported devices, We support iOS 13 and above, and Android 10 and above. 
+We are automatically enabling these Accessibility tools for you with a single click without going to the OS setting!
+
+Follow these steps:
 
 1. **Open a Live Testing Session**: Start any Live Testing session, whether it's a Cross Browser or Native Application test.
 
-2. Access device **Settings > Accessibility**.
+2. Open the **Device Settings** from the left side toolbar.
 
-3. Select **SL_TalkBack Application**: Look for the SL_TalkBack application featuring the Sauce Labs logo.
+3. Select and toggle **Talkback** or **Voiceover**
 
-4. Click **User Service**.
+4. Skip with OK or Cancel the native Talkback/Voiceover guide.
 
-5. Enable **Allow Talkback**.
+5. On Android you can use the Arrow keys (Up-Down-Left-Right) to navigate through the elements, and by hitting the ENTER key, you are able to activate a specific element. 
```

Comment:
should we say  " key modifier shortcuts" are not supported at the moment.

Reply:
i would only call out what is supported. 
We will improve it with the UX, so it is clear what do we support